### ISSUE TITLE
Boolean logic (and, or, and xor) doesn't work in IE

### ISSUE THREAD

**Issue by kethinov** [kethinov/teddy#23]
This is a browser bug.

IE's DOMParser's parseFromString method rearranges attribute order which messes up the whole feature.

They consider this a feature, not a bug, despite every other browser doing the opposite:
http://stackoverflow.com/questions/4445153/javascript-parsefromstring-changing-order-of-elements-in-xml-doc

Regardless, I reported it to Microsoft as a bug here: https://connect.microsoft.com/IE/feedback/details/885393/domparsers-parsefromstring-method-incorrectly-reorders-attribute-order

A possible workaround could involve using regex to detect and preserve the attribute order for conditional elements prior to parsing, but this seems like an expensive solution and a lot of work for what I personally consider a low priority bug since it only affects this feature and only on IE.

I'm open to suggestions for other possible workarounds to try.

**Comment by kethinov** [comment]
Blocked by https://github.com/kethinov/teddy/issues/32
